### PR TITLE
Updated Dutch translations

### LINKED ISSUE #28: Dutch translation needed for UI
The Dutch translation of the user interface has been somewhat left behind for a while.

Is there someone wanting to give a hand with it?

Here is the file to work on: https://github.com/DarwinNE/FidoCadJ/blob/master/bin/MessagesBundle_nl.properties

Have a look at the end of the file, the untranslated resources are in English.

### PR DESCRIPTION
**Pull request by lucaslampe**
Fixes #28

### CODE REVIEW

**Comment by DarwinNE** [comment]
Hi @lucaslampe,
  thank you very much! Your contribution is truly useful since the last Dutch translation of the interface was done a few years ago.

I have just a question: in some resources, I noticed that you removed the spaces at the end of the line. This is mandatory for the Java code, but for the text resources in the interface sometimes the spaces are useful and should be kept in place. For example, for resource "Open_error" the space is needed after the ":" since the file name is added after it. Therefore, without a space the resource may be rendered as follows:
 
    Fout openen bestand:file.fcd

instead of:

    Fout openen bestand: file.fcd

where the latter would be more correct and aestetically pleasing.
Did you remove the spaces by purpose or it was a feature of your editor? Can you please restore those spaces (if their rendering in Dutch is correct, of course).

Kind regards and thanks again!

Davide

**Comment by lucaslampe** [comment]
Woops, no idea how this happend. 😅
I've fixed all of them now.

File: bin/MessagesBundle_nl.properties [hunk 1]
```diff
@@ -100,20 +100,20 @@ Big = Groot
...
-Format_invalid=Ik kan niet een nummer niet herkennen.
-TextToolbar=Laat de tekst labels zien in de toolbar
-SmallIcons=Gebruik kleine icoontjes in de toolbar
-strict_FC_comp=Strikte FidoCad compatibiliteit
-extFCJ_s=Activeer FidoCadJ extenties tijdens het opslaan
-extFCJ_c=Activeer FidoCadJ extenties tijdens kopiëren elementen
-Restart=Opstarten
-Drawing=Circuit
-FidoCad=FidoCadJ extenties
+Format_invalid = Ik kan niet een nummer niet herkennen.
```
Comment (DarwinNE) [inline bin/MessagesBundle_nl.properties]: Hi, here I think that the spaces does not play a very important role

File: bin/MessagesBundle_nl.properties [hunk 2]
```diff
@@ -53,14 +53,14 @@ Circ_opt = Opties
...
 View = Bekijk
 Export_completed = Exporteren is voltooid.
-Export_error = Er zijn fouten gemaakt met exporteren: 
+Export_error = Er zijn fouten gemaakt met exporteren:
 PDF_Font_error = Somige lettertypes zijn niet geëxporteerd. Gebruik Helvetica, Courier, Times of Symbol.
 Illegal_filename = Onjuiste bestandsnaam.
 File_not_found = Bestand niet gevonden.
 Print_uncomplete = Printen is niet volbracht.
-Open_error = Fout openen bestand: 
+Open_error = Fout openen bestand:
```
Comment (DarwinNE) [inline bin/MessagesBundle_nl.properties]: Should not be present a space at the end of the resource, just after the ":"?
It is present in the original resource and it is the last character of the line. Check that your editor does not remove it automatically.

**Comment by DarwinNE** [comment]
Hi @lucaslampe 
No problem at all, just check for the other resources. There are a few of them where there is a space in the last character of the line. You should probably do not remove it!
Open_error is an example, but there are other resources. Check the diffs!

Cheers,

Davide

**Pull request merged**